ip_validation/ip_validation_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rspec'
+require_relative 'ip_validation'
+
+describe '#ipv4?' do
+  describe 'for valid addresses' do
```

Comment:
`nitpick` probably in this case is good to use `context`. [betterspecs](http://www.betterspecs.org/#contexts), 

(same with for `for invalid addresses`)

Reply:
Done